packages/block-library/src/image/edit.js

```diff
@@ -170,10 +195,30 @@ class ImageEdit extends Component {
 			isEditing: false,
 		} );
 
+		const blockWidth = getBlockWidth();
+		let src = media.url;
+		let img = {};
+		let actualWidth;
+		let actualHeight;
+
+		if ( media.sizes ) {
+			// The "full" size is already included in `sizes`.
+			img = media.sizes.large || media.sizes.full;
+			src = img.url;
+			actualWidth = get( img, [ 'actual_size', 'width' ] ) || img.width;
+			actualHeight = get( img, [ 'actual_size', 'height' ] ) || img.height;
```

Comment:
`get()` has a built-in default parameter for the case where the value doesn't exist or is `undefined`. I'm not sure if we'd expect to get `actual_size.width === 0` here but if we don't then we can use the default param.

```js
get( img, 'actual_size.width', img.width );
get( img, 'actual_size.height', img.height );
```

Reply:
Actually seems we will need to take that one step further. Need to ensure both `get( img, [ 'actual_size', 'width' ] )` and `get( img, [ 'actual_size', 'height' ] )` are present before using them, or fall back to `img.width` and `img.height`. The image meta in the DB can be really inconsistent in some rare cases like old plugins mangled it, etc.

Reply:
> Actually seems we will need to take that one step further. Need to ensure both `get( img, [ 'actual_size', 'width' ] )` and `get( img, [ 'actual_size', 'height' ] )` are present before using them, or fall back to `img.width` and `img.height`. The image meta in the DB can be really inconsistent in some rare cases like (old) plugins mangled it, etc.

Condition enforced as of 4d2d959.